### DISCUSSION TITLE
Welcome to msk-markdown Discussions!

### DISCUSSION THREAD

**Discussion by kevinmahrous** [msk-apps/msk-markdown#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DCCICoder** [comment]
Hello, Mr. Kevin

**Reply by kevinmahrous** [reply]
Hi!

**Reply by DCCICoder** [reply]
I 'm a Hardware & Networking Professional willing to change my Career to being a Developer and I know HTML, CSS, JavaScript, C#.NET, SQL and I can even work with Figma or Adobe XD But I don't know why I'm struggling for building a relevant project, Could You Please Help Me through your Guidance... Thanks Mr. Kevin....
